presto-native-execution/src/test/java/com/facebook/presto/nativeworker/ContainerQueryRunnerUtils.java

```diff
@@ -114,9 +117,37 @@ public static void createCoordinatorConfigProperties(int port)
         createPropertiesFile("testcontainers/coordinator/etc/config.properties", properties);
     }
 
-    public static void createCoordinatorJvmConfig()
+    public static void createFunctionNamespaceRemotePropertiesWithFunctionServer(int functionServerPort)
```

Comment:
This is a very long function name createFunctionServerRemoteProperties is reasonable enough.

Please try to reduce other function names as well.